models/labels/social/persona/ens/labels_three_letter_ens_count.sql

```diff
@@ -0,0 +1,36 @@
+{{config(alias='three_letter_ens_count')}}
+
+WITH three_letter_ens_count AS (
+    SELECT 
+        owner,
+        count(owner) as ens_count
+    FROM
+        {{ ref('ens_view_registrations') }}
+    WHERE
+        length(name) = 3
+
+    GROUP BY owner
+    ORDER BY ens_count DESC
+),
+
+top as (
+    SELECT
+        owner
+    FROM
+        three_letter_ens_count
+    LIMIT 1
+)
+
+SELECT
+    'blockchain' as blockchain,
+    (CONCAT('0x', substring(cast(owner as string), 3))) as address,
+    'most_three_letter_ens_owner' as model_name,
+    'spanish-or-vanish' as contributor,
+    'query' as source,
+    'social' as category,
+    timestamp('2022-03-03') as created_at,
+    now() as updated_at,
+    'personas' as label_type,
+    'The owner of the most three letter ENS Domains'as name
+FROM top
+WHERE owner is not null
```

Comment:
Maybe we can remove top CTE and rewrite name feild as
`concat('Number of three letter ENS Domains owned', ens_count) as name`
What do you think @CreedsCode 

```suggestion
SELECT
    'blockchain' as blockchain,
    (CONCAT('0x', substring(cast(owner as string), 3))) as address,
    'most_three_letter_ens_owner' as model_name,
    'spanish-or-vanish' as contributor,
    'query' as source,
    'social' as category,
    timestamp('2022-03-03') as created_at,
    now() as updated_at,
    'personas' as label_type,
    concat('Number of three letter ENS Domains owned', ens_count) as name
FROM three_letter_ens_count
WHERE owner is not null
```